src/components/VLogoLoader/VLogoLoader.types.js

```diff
@@ -0,0 +1,9 @@
+export const propTypes = {
+  status: {
+    type: /** @type {'loading'|'idle'} */ String,
```

Comment:
To get a type cast to work correctly in JSDoc TypeScript you have to wrap the thing you're casing in parenthesis:

```suggestion
    type: /** @type {'loading'|'idle'} */ (String),
```